week-6/Homework/mandatory/3-slideshow/style.css

```diff
@@ -1 +1,11 @@
-/** Write your CSS in here **/
+.jumbotron {
+    padding: 1em;
+    margin: 100px auto auto auto;
+    width: 900px;
+    align-content: center;
+    text-align: center;
+}
+.lead{
+    font-weight: 400;
+    font-size: 25px;
+}
```

Comment:
Nice design! Good to see that you used bootstrap instead of writing everything from scratch. A nice example of how to reuse things 👍